pages/common/lab.md

```diff
@@ -0,0 +1,19 @@
+# lab
+
+> A CLI for Gitlab.
+
+- Create a merge request and specify remote and target branch:
+
+`lab mr create {{remote}} {{target_branch}}`
+
+- The `lab` CLI can call into `hub` commands. Create a pull request on a github repo:
+
+`lab pull-request`
```

Comment:
Do we not need to specify the remote branch ?

Reply:
Nah, `hub` matches from currently checked out branch and targets `origin/master` by default

Reply:
Ok, we should mention that then to be clear. It is not immediately apparent.